### PR TITLE
Support AVITI protocol

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by chuan-wang** [comment]
**NOTE:** The step IDs are based on the values in LIMS-stage. When we migrate to LIMS-production these IDs need to be updated.

**Review by parlundin — APPROVED** [review]

File: LIMS2DB/flowcell_sql.py [hunk 1]
```diff
@@ -2,80 +2,100 @@
...
+
+    if pc_cg.SEQUENCING.get(str(pro.typeid), "") in [
+        "AUTOMATED - NovaSeq Run (NovaSeq 6000 v2.0)",
+        "Illumina Sequencing (NextSeq) v1.0",
+        "MiSeq Run (MiSeq) 4.0",
+        "NovaSeqXPlus Run v1.0",
+    ]:
+        dbname  = "x_flowcells"
+    elif pc_cg.SEQUENCING.get(str(pro.typeid), "") in ["AVITI Run v1.0"]:
+        dbname = "element_runs"
```
Comment (alneberg) [inline LIMS2DB/flowcell_sql.py:86]: Nice!

**Review by alneberg — APPROVED** [review]
Wow,  a lot of changes! I think I managed to find the relevant ones though and they look good!

Nice to stop searching through the old flowcells database as well!

File: scripts/mail_resp.py [hunk 2]
```diff
@@ -11,171 +10,258 @@
...
+    proj_id_view = statusdb["projects"].view("project/project_id")
 
-    operator="par.lundin@scilifelab.se"
-    summary={}
-    project_types=['Bcl Conversion & Demultiplexing (Illumina SBS) 4.0','Illumina Sequencing (Illumina SBS) 4.0',
-    'MiSeq Run (MiSeq) 4.0','Cluster Generation (Illumina SBS) 4.0','Denature, Dilute and Load Sample (MiSeq) 4.0', 'Aggregate QC (DNA) 4.0','Aggregate QC (RNA) 4.0', 'Project Summary 1.3']
+    operator = "par.lundin@scilifelab.se"
+    summary = {}
+    project_types = [
+        "Bcl Conversion & Demultiplexing (Illumina SBS) 4.0",
```
Comment (aanil) [inline scripts/mail_resp.py:26]: Should we be adding all the sequencing steps here? @chuan-wang @chuan-wang ? Do you guys know if people actually look at these emails?

Comment (chuan-wang) [reply scripts/mail_resp.py:26]: I will add it here

**Review by aanil — APPROVED** [review]
Looks good!

**Pull request merged**